src/backend/base/langflow/components/helpers/message_store.py

```diff
@@ -7,11 +7,11 @@
 from langflow.utils.constants import MESSAGE_SENDER_AI, MESSAGE_SENDER_NAME_AI
 
 
-class StoreMessageComponent(Component):
-    display_name = "Store Message"
+class MessageStoreComponent(Component):
+    display_name = "Message Store"
     description = "Stores a chat message or text into Langflow tables or an external memory."
-    icon = "save"
-    name = "StoreMessage"
+    icon = "message-square-text"
+    name = "MessageStore"
```

Comment:
This name can't be changed either

Reply:
Got it. Is it a problem to change the Component Class as well? (MessageStoreComponent)

Reply:
If you set the `name` it ignores the class name. By default `__class__.__name__` is used. Only older components require us to set `name` for backwards compatibility.